### PR TITLE
Cult rework

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
This is a pretty significant Blood Cult overhaul. 

## Why It's Good For The Game
https://hackmd.io/@Xander3359/Skq-NfgGi
Blood Cult needs significant changes before it can become a positive aspect in the game.

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/cult.dm [hunk 1]
```diff
@@ -3,30 +3,40 @@
 #define CULT_LOSS 0
 #define CULT_NARSIE_KILLED -1
 
-/datum/antagonist/cult
+/// Helper to format the text that gets thrown onto the blood storage hud element.
+#define FORMAT_STORED_BLOOD_TEXT(charges) MAPTEXT("<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#dd66dd'>[round(stored_blood)]</font></div>")
```
Comment (san7890) [inline code/modules/antagonists/cult/cult.dm:7]: remember to undefine this

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -285,11 +285,12 @@
...
-		for(var/datum/action/innate/cult/blood_magic/BM in M.actions)
-			to_chat(M, span_cultlarge("Your blood rites falter as holy water scours your body!"))
-			for(var/datum/action/innate/cult/blood_spell/BS in BM.spells)
-				qdel(BS)
+	/// Commenting out Holy Water negating cult spells, will be changed functionality later
+	/* if(IS_CULTIST(M))
+		for(var/datum/action/innate/blood_cult/blood_magic/BM in M.actions) /// Removes the entire list of spells you can prepare as a cultist
+			/// to_chat(M, span_cultlarge("Your blood rites falter as holy water scours your body!"))
+			/// for(var/datum/action/innate/blood_cult/blood_spell/BS in BM.spells)
+				qdel(BS) */
```
Comment (dragomagol) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:293]: I know this is a WIP but don't forget to remove code instead of commenting it out

**Comment by Mothblocks** [comment]
firefox_2022-10-03T19-48-42
firefox_2022-10-03T19-48-47

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 2 above)
Comment (Xander3359) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:293]: oh yeah, I have a notepad specifically with all the comments that I can copy/paste once I'm ready to revisit them.

**Comment by Mothblocks** [comment]
i read your doc and its just cult but theres more complicated stuff in other antagonists??

**Comment by Jackal-boop** [comment]
Xander is based he can make this work

**Comment by Mothblocks** [comment]
i am inches away from rage closing this so im going to wait a few days and see if im insane or if this is just cult but theres more garbage

**Comment by Xander3359** [comment]
Currently this is just reworking cult. I'd rather get this done before even thinking of beginning to work on clock cult. A lot of the problems with cult stem from it being overfilled with bloat, which makes it so most players that are new to cult have no clear indication of what they should be doing (other than murderbone).

**Comment by Mothblocks** [comment]
so what does *this* pr do

**Comment by Pumpkinoe** [comment]
I don't really get it - the design doc talks about reworking cult so that there's clock cult and stuff, but then you say you don't want to even think about reworking clockcult at the moment? Very confusing.

**Comment by Xander3359** [comment]
This PR is a WIP, but it's intended to rework blood cult. Currently many people are confused on the cult mechanics, all the different spells and runes tend to overwhelm people, and most go unused. There is no clear way to fight as/against them, combat ends up being unfun because it takes a single click from a cultist stun spell to basically end the encounter. I also personally feel as if blood cult is very far from it's theme, I believe my changes moves it more towards a thematic "blood cult", where it harvests blood to use their powers, and enourages more bloodshed by making convert rune work on dead people.

**Comment by Mothblocks** [comment]
BUT WAHT DOES IT DO

**Comment by Mothblocks** [comment]
*this* pr what does it do *exactly*

**Comment by Xander3359** [comment]
ah, currently I've implemented the changes to conversion rune, adds blood storage (with placeholder changeling UI), makes the spells available by default to cultists  and makes them cost blood. It also cut out a lot of bloat (most runes are gone, most spells are gone). It's WIP, so future plans involve adding a gear rune, construct rune, making a structure that dispenses huds, water, sharpener, shuttle curse. Should I list all this in the changelog?

**Comment by Xander3359** [comment]
> I don't really get it - the design doc talks about reworking cult so that there's clock cult and stuff, but then you say you don't want to even think about reworking clockcult at the moment? Very confusing.

I'm reworking blood cult, clock cult is future plans.

**Comment by Mothblocks** [comment]
> Should I list all this in the changelog?

Yes, and your why it's good for the game, detailing why each change is good. Right now I have utterly no clue what this is actually doing

I will warn you it's extremely unlikely this gets merged as is but I'm just so utterly confused that I can't even close it yet

**Comment by Xander3359** [comment]
Would it be better if I were to close this and re-open a PR once it's no longer WIP? That way I can get a complete changelog setup in a single PR.

**Comment by Mothblocks** [comment]
Yes

**Comment by Xander3359** [comment]
Ok, I will be closing this PR, I will be re-opening it when I am more satisfied with my changes and feel like it's in a better state that it could possibly be test merged (as well as a proper documented changelog). Thanks for all the feedback.

**Pull request closed without merging**